packages/wonder-blocks-progress-spinner/__snapshots__/generated-snapshot.test.js.snap

```diff
@@ -49,7 +49,7 @@ exports[`wonder-blocks-progress-spinner example 1 1`] = `
     <path
       className=""
       d="M88.38 46.91a3.818 3.818 0 1 1 7.602 0h.006c.008.362.012.725.012 1.09 0 26.51-21.49 48-48 48S0 74.51 0 48 21.49 0 48 0c.365 0 .728.004 1.09.012v.005a3.818 3.818 0 1 1 0 7.602v.032c-.362-.01-.725-.015-1.09-.015C25.708 7.636 7.636 25.708 7.636 48c0 22.292 18.072 40.364 40.364 40.364 22.292 0 40.364-18.072 40.364-40.364 0-.365-.005-.728-.015-1.09h.032z"
-      fill-rule="nonzero"
+      fillRule="nonzero"
```

Comment:
Updated the snapshot for snapshots tests since this change caused the existing snapshot tests to fail. You can see that the only change is the `fill-rule` -> `fillRule` change.